Don't hide cursor on modifier press (#3661)
alacritty/alacritty: CHANGELOG.md

- Preserve selection on non-LMB or mouse mode clicks
- Wayland client side decorations are now based on config colorscheme
- Low resolution window decoration icon on Windows

> - Don't hide cursor on modifier press (mouse.hide_when_typing setting)

Reviewer: ```suggestion
- Don't hide cursor on modifier press with `mouse.hide_when_typing` enabled
```

This is about the same length but reads a bit nicer I think.
